imap_processing/swe/l1b/swe_l1b_science.py

```diff
@@ -0,0 +1,509 @@
+import dataclasses
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+from imap_processing import imap_module_directory
+from imap_processing.cdfutils.global_base import Epoch
+from imap_processing.swe import swe_cdf_attrs
+
+# ESA voltage and index in the final data table
+esa_voltage_row_index_dict = {
+    0.56: 0,
+    0.78: 1,
+    1.08: 2,
+    1.51: 3,
+    2.10: 4,
+    2.92: 5,
+    4.06: 6,
+    5.64: 7,
+    7.85: 8,
+    10.92: 9,
+    15.19: 10,
+    21.13: 11,
+    29.39: 12,
+    40.88: 13,
+    56.87: 14,
+    79.10: 15,
+    110.03: 16,
+    153.05: 17,
+    212.89: 18,
+    296.14: 19,
+    411.93: 20,
+    572.99: 21,
+    797.03: 22,
+    1108.66: 23,
+}
+
+
+def read_lookup_table(table_index_value: int):
+    """Read lookup table from file.
+
+    Parameters
+    ----------
+    table_index_value : int
+        ESA table index number
+    """
+    lookup_table_filepath = f"{imap_module_directory}/swe/l1b/swe_esa_lookup_table.csv"
```

Comment:
Not critical, but you could keep it as a Path object here instead of string.
```suggestion
    lookup_table_filepath = imap_module_directory / "swe/l1b/swe_esa_lookup_table.csv"
```